lib/ex_sieve/builder.ex

```diff
@@ -20,10 +20,12 @@ defmodule ExSieve.Builder do
     do: OrderBy.build(query, sorts, binding)
 
   defp build_binding(relations) do
-    relations
+    bindings = relations
     |> List.insert_at(0, :query)
     |> Enum.map(&Macro.var(&1, Elixir))
-    |> Builder.escape_binding
+    {_, vars} = Builder.escape_binding(%Ecto.Query{}, bindings)
```

Comment:
you can pass the query, to buind_binding. e.g:
```
  defp build_binding(query, relations) do
    bindings = relations
    |> List.insert_at(0, :query)
    |> Enum.map(&Macro.var(&1, Elixir))

    Builder.escape_binding(query, bindings)
  end
```
and extract this in call function:
```
  defp build_binding(query, relations) do
    bindings = relations
    |> List.insert_at(0, :query)
    |> Enum.map(&Macro.var(&1, Elixir))

    Builder.escape_binding(query, bindings)
  end
```